sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -314,4 +323,28 @@ private PlaySourceInternal translatePlaySourceToPlaySourceInternal(PlaySource pl
         }
         return playSourceInternal;
     }
+
+    private List<RecognizeChoice> translateRecognizeChoiceListtoInternal(List<com.azure.communication.callautomation.models.RecognizeChoice> recognizeChoices) {
```

Comment:
To avoid confusion, this should be called RecognizeChoiceInternal.

Reply:
do you mean the function name?